guide/interactions/registering-slash-commands.md

```diff
@@ -0,0 +1,192 @@
+# Registering slash commands
+
+Discord provides users with the option to create client-integrated slash commands. In this section, you'll be learning how to register these commands using discord.js!
+
+::: tip
+If you already have slash commands set-up for your application and want to learn to respond to them, refer to [the following page](/interactions/replying-to-slash-commands/).
+:::
+
+
+## Global commands
+
+First up, we'll introduce you to global application commands. These types of commands will be available in all guilds your application has the `applications.commands` scope authorized, as well as in DMs.
+
+::: tip
+Global commands are cached for one hour. That means that new global commands will fan out slowly across all guilds and will only be guaranteed to be updated in an hour. Guild commands however update instantly. As such, we recommend you use these for quick testing and global commands when they're ready for public use.
+:::
+
+So, to register a global command we'll be passing an `ApplicationCommandData` object to the `ApplicationCommandManager#create()` method as follows:
+
+```js
+client.on('message', async message => {
+	if (!client.application?.owner) await client.application?.fetch();
+
+	if (message.content.toLowerCase() === '!deploy' && message.author.id === client.application?.owner.id) {
+		const data = {
+			name: 'ping',
+			description: 'Replies with Pong!',
+		};
+
+		const command = await client.application?.commands.create(data);
+		console.log(command);
+	}
+});
+```
+::: danger
+Command names must be lowercase at all times or else you will receive an API error.
```

Comment:
```suggestion
Command names must be lowercase. You will receive an API error if this is not the case.
```

Reply:
You will receive an API error otherwise.

Reply:
> Command names must be lowercase. You will receive an API error if this is not the case.

I think `You will receive an API error otherwise.` is better here.